contracts/custom/DecentDAOVoting.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity =0.8.19;
+
+import { LinearERC20Voting } from "../azorius/LinearERC20Voting.sol";
+import { VotesERC20Wrapper } from "../VotesERC20Wrapper.sol";
+
+interface Escrow {
+    function getBalance(address _address, uint _block) external;
+}
+
+contract DecentDAOVoting is LinearERC20Voting {
+
+    Escrow public escrow;
+
+    // TODO override setUp() function to initialize the escrow contract
```

Comment:
the address of the escrow contract would be supplied in setup()

Reply:
do you plan to address this TODO in this PR or another one?

Reply:
yeah I'll wrap that up next week, I wanted to discuss the setup with @tbwebb22

Reply:
Hmm so the `setUp` function in `LinearERC20Voting` would also need to be changed to `virtual`